cost-analyzer/values.yaml

```diff
@@ -419,10 +420,13 @@ serviceAccount:
 #  azureSubscriptionID: 0bd50fdf-c923-4e1e-850c-196dd3dcc5d3
 #  azureClientID: f2ef6f7d-71fb-47c8-b766-8d63a19db017
 #  azureTenantID: 72faf3ff-7a3f-4597-b0d9-7b0b201bb23a
+#  azureClientPassword: fake key # Only use if your values.yaml are stored encrypted. Otherwise provide an existing secret via serviceKeySecretName
 #  discount: "" # percentage discount applied to compute
 #  negotiatedDiscount: "" # custom negotiated cloud provider discount
 #  defaultIdle: false
-#  serviceKeySecretName: ""
+#  serviceKeySecretName: "" # Use an existing AWS or Azure secret with format as in aws-service-key-secret.yaml or azure-service-key-secret.yaml 
+#  createServiceKey: true # Creates a service key 
+#  createdServiceKeyType: AZURE # AWS or AZURE
```

Comment:
is this field necessary?

Reply:
Yeah, unfortunately since the keys have different formats.

Reply:
If we are creating the secret, can we not detect if either an AWS or Azure has been supplied and then use the proper format?

Reply:
Oh, yeah fair point. I'll fix

Reply:
Ready for another look.